zha/zigbee/cluster_handlers/__init__.py

```diff
@@ -441,6 +444,10 @@ async def async_configure(self) -> None:
             if ch_specific_cfg:
                 self.debug("Performing cluster handler specific configuration")
                 await ch_specific_cfg()
+
+            self.debug("Discovering available attributes")
+            await self.discover_unsupported_attributes()
```

Comment:
nit: the name of the method and the comment as written are opposites (available vs unsupported)